### PR TITLE
Upgrade Longhorn engine integration tests to use python 3

### PR DESCRIPTION
**Pull request by meldafrawi**

### CODE REVIEW

File: integration/data/snapshot_tree.py [hunk 1]
```diff
@@ -63,55 +63,77 @@ def snapshot_tree_verify_relationship(address, snap, strict):
     info = cmd.snapshot_info(address)
 
     assert snap["0a"] in info
-    assert snap["0b"] in info[snap["0a"]]["children"]
+    assert snap["0b"] in \
```
Comment (yasker) [inline integration/data/snapshot_tree.py]: We no longer need to change those, right?

File: integration/data/test_basic_ops.py [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
 
-import cmd
-from common import (  # NOQA
-    dev, get_dev, read_dev, write_dev,  # NOQA
-    grpc_engine_manager,  # NOQA
-    grpc_controller, grpc_replica1, grpc_replica2,  # NOQA
+import data.cmd as cmd
+from data.common import (  # NOQA
+    get_dev, read_dev, write_dev,  # NOQA
+    #grpc_engine_manager,  # NOQA
```
Comment (yasker) [inline integration/data/test_basic_ops.py]: unused? Then we can remove them.

Comment (meldafrawi) [reply integration/data/test_basic_ops.py]: fixed

File: integration/data/snapshot_tree.py (hunk 1 above)
Comment (meldafrawi) [reply integration/data/snapshot_tree.py]: fixed

**Pull request merged**